.gitignore

```diff
@@ -25,4 +25,5 @@ target/
 .vscode/
 .mvn/
 mvnw
-.idea
\ No newline at end of file
+.idea
```

Comment:
is this change required?

Reply:
yes, otherwise someone might commit the useless files by mistake